app/src/main/AndroidManifest.xml

```diff
@@ -25,43 +26,50 @@
         </activity>
 
         <activity
+            android:exported="false"
```

Comment:
This is not required - if activity has no intent-filter the default value would be false.

Reply:
done